### PR TITLE
Feature/upload files to service

### PR DESCRIPTION
**Pull request by nattravn**
You can choose where you want upload local files "Bruse", "Dropbox" or "Google Drive"

### CODE REVIEW

**Comment by ronjagrosz** [comment]
It would be nice to create a folder in dropbox and drive that the files uploads to.

**Comment by tkomstadius** [comment]
it would be good to have a default identity when saving

File: app/views/files/files/show_all.html.erb [hunk 1]
```diff
@@ -10,6 +10,13 @@
   <div class="actions">
     <%= f.submit "Upload" %>
   </div>
+
+  <% current_user.identities.each do |identity| %>
+    <%= radio_button_tag(:service, identity.id) %>
```
Comment (danielronnkvist) [inline app/views/files/files/show_all.html.erb]: Make sure either the `default_identity_id` or some other is checked from start.

File: db/schema.rb [hunk 2]
```diff
@@ -81,7 +81,6 @@
     t.datetime "confirmation_sent_at"
     t.string   "unconfirmed_email"
     t.boolean  "own_password",           default: true
-    t.integer  "default_identity_id"
```
Comment (danielronnkvist) [inline db/schema.rb:84]: Ooops, we need this one!

**Comment by tkomstadius** [comment]
I don't get files uploaded to Drive, it says that I must have a local account

**Comment by tkomstadius** [comment]
If i press upload without adding a file I get: "undefined method `[]' for nil:NilClass". We should probably have some kind of catch for that. :)

**Comment by tkomstadius** [comment]
Otherwise I think it looks good :cactus:

File: app/controllers/files/browse_controller.rb [hunk 3]
```diff
@@ -22,21 +22,37 @@ def browse
 
 
   def upload
-    uploader = LocalFileUploader.new
+    identity = Identity.find(params[:service])
+    if(params[:file].blank?)
```
Comment (danielronnkvist) [inline app/controllers/files/browse_controller.rb]: Shouldn't this be `params[:bruse_file]`?

Comment (ronjagrosz) [inline app/controllers/files/browse_controller.rb]: Yes!

**Comment by danielronnkvist** [comment]
When files are uploaded to Dropbox or Drive, they aren't saved in the database as BruseFiles. :city_sunrise:

**Comment by danielronnkvist** [comment]
I have fixed so the files are stored in the database, someone have to check that the google drive files are stored as well. :cat2:

**Comment by ronjagrosz** [comment]
Seems to work with drive as well :cat:

**Pull request merged**